tensorflow_addons/image/filters.py

```diff
@@ -112,6 +109,66 @@ def mean_filter2d(image, filter_shape=(3, 3), name=None):
         return y
 
 
+@tf.function
+def median_filter1d(image,filter_shape=3, name=None):
+    """This method performs 1D Median Filtering on images.Filter shape can be user given.
+       This method takes both kind of images where pixel values lie between 0 to 255 and where it lies between 0.0 and 1.0
+       Args:
+           input: A 3D `Tensor` of type `float32` or 'int32' or 'float64' or 'int64 and of shape`[rows, columns, channels]`
+
+           filter_shape: Optional Argument.Single Integer. Default value = 3
+
+        Returns:
+            A 1D median filtered image 3D tensor of shape [rows,columns,channels] and type 'int32'. Pixel value of returned 
+            tensor ranges between 0 to 255
```

Comment:
should be `backticks` instead of 'quotation marks' if you want to highlight the word in docstring. Also, some spaces between words and punctuations are missing. Please check it.

Reply:
I don't want to highlight any word. Solved.

Reply:
Ok, but still adreess some issues of spaces

1. 
This method performs 1D Median Filtering on images.Filter shape can be user given.
=>
This method performs 1D Median Filtering on images. Filter shape can be user given.

2. 
input: A 3D `Tensor` of type `float32` or 'int32' or 'float64' or 'int64 and of shape`[rows, columns, channels]`
=>
input: A 3D `Tensor` of type `float32` or 'int32' or 'float64' or 'int64 and of shape `[rows, columns, channels]`

3.
filter_shape: Optional Argument.Single Integer. Default value = 3
=>
filter_shape: Optional Argument. Single Integer. Default value = 3

4.
A 1D median filtered image 3D tensor of shape [rows,columns,channels]
=>
A 1D median filtered image 3D tensor of shape [rows, columns, channels]

Reply:
@Mainak431 Please address the comments of the reviewers. The PR reviewers get to decide when an issue is resolved.

Reply:
Ok . @WindQAQ Now i got what you mean by issue of spaces.

Reply:
@seanpmorgan Yes of course.